### ISSUE TITLE
Linux meson builddir error

### ISSUE THREAD

**Issue by sabinagreen** [basler/gst-plugin-pylon#20]
I'm trying to build pylonsrc plugin in my Linux machine (ubuntu 18, x86_64) and face some problems.
 
my steps are:

I installed Basler ver 7.1.0 
 
sudo apt remove meson ninja-build
sudo -H python3 -m pip install meson ninja --upgrade
sudo apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev cmake
export PYLON_ROOT=/opt/pylon
meson builddir --prefix /usr/

 error: there is no messon.build directory
 
could you pls help with it?

**Comment by michaelgruner** [comment]
Thanks for your report, those steps look correct. Can you share the full log?

**Comment by sabinagreen** [comment]
pls see it.
[meson-log.txt](https://github.com/basler/gst-plugin-pylon/files/9918069/meson-log.txt)

**Comment by thiesmoeller** [comment]
.. are you really in this project's source dir when you run `meson build` ?

**Comment by sabinagreen** [comment]
yes, I'm running it from messon directory

**Comment by thiesmoeller** [comment]
Hi @sabinagreen,

Looking at the attached log file:
Have you checked out this project *gst-plugin-pylon* into a folder named *gst* ?

In your log file is the line 
```
Source dir: /home/lumenis/gst
```

And some APP_VERSION = 0.7.5 which is not from this project's code. 

Most probably this project here: 

https://github.com/leinardi/gst

Can you please make 

* a fresh clone of gst-plugin-pylon in some different directory
* `cd gst-plugin-pylon`
* follow the readme to build on linux

**Comment by sabinagreen** [comment]
Thanks, now it work :)!
One additional question: gst-inspect-1.0 pylonsrc
 work only with parameter --gst-plugin-path=/usr/lib/x86_64-linux-gnu/gstreamer-1.0.

**Comment by thiesmoeller** [comment]
can you please post the output of:

```bash
gst-inspect-1.0 fakesink | grep Filename
```

This will list the install path of a standard plugin.. 

From all what you describe, it seems that you are not running in a standard Ubuntu environment.....

**Comment by thiesmoeller** [comment]
documentation fix in v0.5.0
